tests/test_finder.py

```diff
@@ -94,7 +94,7 @@ def test_editables_file_handling(tmp_path):
     """
     pth_file = tmp_path / "__editables_whatever.py"
     pth_file.write_text("hello\nF.map_module('griffe', 'src/griffe/__init__.py')")
-    assert _handle_editable_module(pth_file) == Path("src")
+    assert list(_handle_editable_module(pth_file)) == [Path("src")]
```

Comment:
`_handle_editable_module` now returns a list, right? If true, then I don't think we need to wrap it in `list` again?